CHANGELOG.txt

```diff
@@ -1,4 +1,8 @@
 releases:
+  v1.20.0:
+  -ssh_key: add use_secret_of_key_name_if_exists property to
+   cloudify.keys.nodes.RSAKey node.
```

Comment:
I think the name is a little confusing. Too many words. How about change it to `use_existing_secret_if_exists`? Or is that also no good?